ecs-init/engine/engine.go

```diff
@@ -168,7 +170,14 @@ func (e *Engine) load(image io.ReadCloser, err error) error {
 // StartSupervised starts the ECS Agent and ensures it stays running, except for terminal errors (indicated by an agent exit code of 5)
 func (e *Engine) StartSupervised() error {
 	agentExitCode := -1
-	for agentExitCode != terminalSuccessAgentExitCode && agentExitCode != terminalFailureAgentExitCode {
+	const (
+		minTime    = time.Millisecond * 500
+		maxTime    = time.Second * 15
+		jitter     = 0.10
+		multiplier = 2.0
+	)
+	backoff0 := backoff.NewBackoff(minTime, maxTime, jitter, multiplier, 99999999)
```

Comment:
what is 99999999?

Reply:
that's the max number of retries, it's not getting used in this case because we don't call "backoff.ShouldRetry" anyways

Reply:
can you either make it a const or add a comment explaining the number?

Reply:
sure will do